FreeOTP/EditViewController.swift

```diff
@@ -158,8 +161,7 @@ class EditViewController : UIViewController, UITextFieldDelegate, UIImagePickerC
             })
         })
 
-        // Why does this cause a runtime error, and what is this even for?
-//        textField(issuer, shouldChangeCharactersIn: NSRange(), replacementString: "")
+        textField(issuer, shouldChangeCharactersIn: NSRange(), replacementString: "")
```

Comment:
So it appears that you removed this line before and then added it back in. Please rework the patches to avoid this code churn.